okio/src/jvmMain/kotlin/okio/internal/ResourceFileSystem.kt

```diff
@@ -157,6 +155,47 @@ internal class ResourceFileSystem internal constructor(
     return canonicalThis.relativeTo(ROOT).toString()
   }
 
+  /**
```

Comment:
These are all moved from the companion object. I moved 'em to get access to the `systemFileSystem` property, which is introduced to make testing easier.